PRODUCTION.md

```diff
@@ -0,0 +1,55 @@
+# Scality S3server in production with Docker
```

Comment:
Using S3 for CI testing is to force hostname and credential using env variables.